### ISSUE TITLE
Development version/release of plots

### ISSUE THREAD

**Issue by emcwhirter-usgs** [usgs/geomag-plots#48]
We need a place to display data that is still in development and testing before releasing it to the main page. This could possibly cover the need for displaying ObsRIO data as well.

**Comment by emcwhirter-usgs** [comment]
Would it be useful to put observatory and channel parameters in a config file so that we/anyone can have multiple deploys for different data?

**Comment by erigler** [comment]
I interpret this feature description to be a "beta" page, which I definitely advocate, especially if it is publicly viewable, if not necessarily publicly advertised. Is this the intent?

I wonder if a different feature is what is needed for displaying engineering data, or any other data that is not necessarily intended for public consumption.

I don't understand your 2nd comment @emcwhirter-usgs. Can you clarify?

**Comment by emcwhirter-usgs** [comment]
Yes, from what we have discussed, the intention is to have a "beta" page, or more accurately a "non-vetted data" page. Basically, a place to put any data that isn't well documented, verified, validated and approved for widespread distribution. This would include ObsRIO data and any other beta products we develop.

My second comment is directed at the development of this feature since channels are currently hard-coded.

**Comment by erigler** [comment]
Am I correct to summarize that, eventually, we want:

1) a mature, public products page;
2) a non-vetted, non-advertised products page;
3) a page dedicated to replacing WebDisplay functionality?

**Comment by emcwhirter-usgs** [comment]
1) Yes
2) Yes
3) Maybe, this may end up being part of 1) with some extra features to logged in users, or it may be its own thing. It is probably ~2 years out, so we haven't really discussed details of what it will look like. We've just started planning some of the data and features that we'll eventually need.
